templates/basic/.github/workflows/publish.yaml

```diff
@@ -21,16 +21,16 @@ jobs:
         uses: "google-github-actions/auth@v2"
         with:
           token_format: access_token
-          workload_identity_provider: ${{ vars.GOOGLE_WORKLOAD_IDENTITY_PROVIDER }}
-          service_account: ${{ vars.GOOGLE_SERVICE_ACCOUNT }}
+          workload_identity_provider: ${{"{{"}} vars.GOOGLE_WORKLOAD_IDENTITY_PROVIDER {{"}}"}}
```

Comment:
I think this is how you're meant to deal with literal `{{` and `}}` in templates. At least I don't see any other solution in the documenation.

Reply:
yeah I think this is it indeed... since we dont need to escape in a lot of places maybe it is not that bad x_x